Common_Words_Scraping.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+import time
+from lxml import html
+import requests
+import re
+from bs4 import BeautifulSoup
+import os
+import csv
+from Wiktionary_scraper import *
+
+def common_words(language):
+    """
+    function that will give us the 1000 most common words in a language from http://1000mostcommonwords.com
+    """
+    url = 'http://1000mostcommonwords.com/1000-most-common-%s-words/' %language
+    try:
+        request = requests.get(url).content
+    except:
+        print("Request failed")
+  
+    soup = BeautifulSoup(request, 'lxml')
+    section = soup.find(class_="entry-content")
+    sub_sections = section.find_all('tr')
+    common_words = []
+    for sub_section in sub_sections:
+        line = sub_section.find_all('td')
+        common_word = line[1].get_text()
+        common_word = common_word.lower()
+        common_words.append(common_word)
```

Comment:
can fit in one line without harming readability: common_words.append(line[1].get_text().lower())